### PR TITLE
Rename HeaderNew -> Header

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Separated folder renames from previous PR to make it easier to review

`HeaderNew` -> `Header`
`HeaderBlockNew` -> `HeaderBlock`

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4737?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4739** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4739?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4738** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4738?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4737** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4737?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4735** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4735?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/app/switcher/layout.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { type ReactNode } from 'react'
 import { TranslationsProvider } from '@/appComponents/providers/TranslationsProvider'
 import { RootLayout } from '@/appComponents/RootLayout/RootLayout'
-import { Header } from '@/components/HeaderNew/Header'
+import { Header } from '@/components/Header/Header'
```
Comment (gustaveen) [inline apps/store/src/app/switcher/layout.tsx:4]: Didn't view this page how it looks with the new header but I suppose it's fine 👍

**Review by gustaveen — APPROVED** [review]
Ah, smart that you did the rename in this PR 👌

**Comment by alebedev** [comment]
### Merge activity

* **Sep 18, 10:25 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4737).

**Pull request merged**